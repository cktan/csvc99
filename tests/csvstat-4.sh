# Test Case : Non-UTF Characters
../csvstat in/csvstat-3.csv 
