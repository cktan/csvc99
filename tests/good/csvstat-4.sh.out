      #bytes: 58
       #rows: 3
    #columns: 2 .. 5
avg row size: 19
min row size: 8
max row size: 27
